src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+}
+
+header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  height: 60px; /* Высота хедера по дизайну */
+}
+
+a:hover {
+  color: #00acdc;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+}
+
+.header_logo {
+  margin: 10px 0 10px 50px;
+}
+
+nav {
+  margin-right: 50px;
+  display: flex;
+  position: relative;
+}
+
+nav ul li {
+  margin-right: 20px;
+  margin-top: 22.5px;
+  margin-bottom: 22.5px;
+}
+
+nav ul li:last-child {
+  margin-right: 0;
+}
+
+nav ul {
+  list-style: none;
+  display: flex;
+  padding: 0;
+  margin: 0;
```

Comment:
<img width="964" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/b275618f-0772-44ff-97d9-87d66afce9c3">

Reply:
Fixed all. Check this commit 
https://github.com/mate-academy/layout_moyo-header/pull/4942/commits/1155635c8fe8ab64f9387242ab3eac58e4b55422